simulate-switching-workspaces-on-active-monitor@micheledaros.com/extension.ts

```diff
@@ -0,0 +1,385 @@
+import Clutter from "gi://Clutter";
```

Comment:
Hi @ProbstDJakob thanks for porting the extension to Typesctipt.
Would you be so kind to update the README with detailed instructions on how to build it?
I've tried on my system (with npm 10.8.1) and got many errors

Reply:
I've created an issue for it: https://github.com/micheledaros/gnome-shell-extension-simulate-switching-workspaces-on-active-monitor/issues/10